modules/cloudwatch-alarm-actions/modules/lambda-subscription/src/jira/lambda.py

```diff
@@ -37,7 +37,7 @@ def create_jira_ticket(summary,description):
         print("Response content:", response.content)
 
 def handler(event, context):
-    file_url = "https://raw.githubusercontent.com/dasmeta/terraform-aws-monitoring/DMVP-2705/modules/cloudwatch-alarm-actions/modules/lambda-subscription/src/event_handler.py"
+    file_url = "https://raw.githubusercontent.com/dasmeta/terraform-aws-monitoring/main/modules/cloudwatch-alarm-actions/modules/lambda-subscription/src/event_handler.py"
```

Comment:
why we use url to import python module/file which is in file/folder next to this one?
why not push this event_handler.py file withing lambda zip file when terraform runs and create lambda?
so that after we you can have this simple form of import in lambda.py, and all the changes in lambda.py and event_handler.py will go along side in same branch:
```python
# in top of lambda.py files
import ./event_handler.py
....
```